server/supernova/CMakeLists.txt

```diff
@@ -106,22 +106,32 @@ else()
   endif()
 endif()
 
-if(APPLE OR MSYS)
+if((APPLE OR MSYS) AND (AUDIOAPI STREQUAL portaudio))
```

Comment:
I think this condition is wrong - should be `(APPLE or (MSYS AND (AUDIOAPI STREQUAL portaudio)))`

Reply:
ok. Solved